src/orchestration/Orchestration.ts

```diff
@@ -215,6 +221,8 @@ export class Orchestration {
 
         if (this.config.enableRumClient) {
             this.enable();
+            // Enable monkey patch to interecept AJAX requests for virtual page load timing
+            this.eventCache.enableVirtualPageLoadTimerPatch();
```

Comment:
suggestion: Since this is always enabled, remove this sequence of calls and instead enable the monkey patch in the `VirtualPageLoadTimer` constructor.